packages/webpack-expose-package-info-plugin/README.md

```diff
@@ -0,0 +1,52 @@
+Expose package information
+======================
+
+This plugin is useful if you want to expose chosen package information to
+a bundle created via webpack.
+
+The mechanism uses the DefinePlugin to define a variable that will contain
+package informations.
+
+### Usage
+
+```
+const PackageInfoPlugin = require('webpack-expose-packages-version')
+
+new PackageInfoPlugin({
+  packages: ['cozy-bar', 'cozy-ui']
+})
+```
+
+#### Exposed variable
+
+By default the exposed variable is __PACKAGES__ but it can be customized
+via the `varName` option.
+
+```patch
+ new PackageInfoPlugin({
+   packages: ['cozy-bar', 'cozy-ui'],
++  varName: 'PACKAGES'
+ })
+```
+
+You can now access package information in your app.
+
+```
+/* global PACKAGES */
+
+console.log(PACKAGES)
+/* {"cozy-bar": {"version": "1.0.0"}, "cozy-ui": {"version": "6.26.2"}} */
```

Comment:
It works on mobile since we build the bar, but it will give us a false information on the web version. In our app, we should take care of that( better to hide a wrong information, right ? :D)

Reply:
That's a good point ! We'll put a conditional in our config.